docs/getting-started/part-1/select-columns.md

```diff
@@ -234,7 +235,7 @@ podman run --rm -v ${PWD}/data:/mnt/data -v ${PWD}/target:/mnt/lib -v ${PWD}/con
 </TabItem>
 </Tabs>
 
-:::warning
+:::danger
```

Comment:
Why this change? Warning is still supported.

Reply:
It seems like the code on the repo is not the same as the code deployed to [smartdatalake.ch](https://smartdatalake.ch/docs/getting-started/part-1/select-columns). On the website, this box is of type "DANGER":
![image](https://github.com/smart-data-lake/smart-data-lake/assets/81236959/7bedca59-2e4a-4e82-8dad-e40db27041d3)
In the image, the older version of the markdown parser in docusaurus 2 has some problems with the quotation. That is fixed with the new version of MDX (markdown parser).